📖 Import windows documentation (#93)
adjust/dev-docs: docs/windows/recording/attribution.md

# Get attribution information

> When a user interacts with a campaign link, their attribution information updates. This can happen if the user interacts with a [deep link](https://help.adjust.com/en/article/deep-links). The SDK can listen for attribution changes and call a function when it detects an update.

Reviewer: ```suggestion
When a user interacts with a campaign link or [deep link](https://help.adjust.com/en/article/deep-links) their attribution information can update. The SDK can listen for attribution changes and call a function when it detects an update.
```

Reviewer: Maybe should qualify that attribution updates arent always guaranteed when a user interacts with a link, since other criteria need to be met too
